src/Feedback-Service/Services/FeedbackService.cs

```diff
@@ -0,0 +1,88 @@
+using Feedback_Service.Dtos;
+using Feedback_Service.Entities;
+using Feedback_Service.Interfaces;
+using Feedback_Service.Models;
+using Feedback_Service.Repository;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Feedback_Service.Services;
+
+public class FeedbackService : IFeedbackService
+{
+    private readonly IFeedbackEntriesRepository feedbackEntriesRepository;
+
+    public FeedbackService(IFeedbackEntriesRepository feedbackEntriesRepository)
+    {
+        this.feedbackEntriesRepository = feedbackEntriesRepository;
+    }
+
+    public async Task<IEnumerable<FeedbackDto>> GetAll()
+    {
+        var feedbackEntries = (await feedbackEntriesRepository.GetAll())
+                                .Select(feedbackEntry => feedbackEntry.AsDto());
+        return feedbackEntries;
+    }
+
+    public async Task<FeedbackDto?> GetFeedbackById(Guid id)
+    {
+        var feedbackEntry = await feedbackEntriesRepository.Get(id);
+
+        if (feedbackEntry == null) return null;
+
+        return feedbackEntry.AsDto();
+    }
+
+    public async Task<IEnumerable<FeedbackDto>> GetPatientFeedbackEntryById(int patientId)
+    {
+        var feedbackEntries = (await feedbackEntriesRepository.GetPatientFeedbacks(patientId))
+                            .Select(feedbackEntry => feedbackEntry.AsDto());
+
+        System.Diagnostics.Debug.WriteLine(feedbackEntries);
```

Comment:
I think this was probably used to debugging. Interesting that you used `System.Diagnostics.Debug`, I have never seen it before, but it's really cool! Apparently it is removed by the compiler on runtime and is a lot faster than `Console.Writeline`, so I'll definitely be stealing this 😉

Reply:
haha thank you and i totally forgot to remove it.